manual/background.md

```diff
@@ -0,0 +1,20 @@
+# Background
+
+LinguaCafe is a self-hosted web-app for learning foreign languages by reading.
+
+In summary, you import foreign language texts into LinguaCafe from ebooks, websites, etc., and read through them, creating saved definitions and associations for words as you read. As you read more, you continue to build your personal dictionary.
+  
+I developed LinguaCafe mainly for personal use, because I found the alternative platforms too expensive or lacking in features I wanted.  After years of development and reading other language learners' opinions about similar platforms, I realized that it might also be a useful tool for other learners, so I have started adding more features to grow LinguaCafe into a platform that more people can benefit from.  
+
+With LinguaCafe you:
+- **Can read the source code and contribute features.** LinguaCafe is an open-source software, which means the code is readily available to you, you can copy it and use it according to the license, and contribute features.
+- **Can self-host.** LinguaCafe is self-hosted, web app which means it runs in your machine or potentially in a private web server and thus be accessible to you in the internet.
+- **Control your data.** Because you host LinguaCage yourself in your machine, you own your data and can do whatever you want with it. 
+- **Free.** LinguaCafe doesn't have a price tag, it is a free as in *free beer*.
+
+
+## LinguaCage alternatives
```

Comment:
Please remove non open-source and free alternatives from here. I do not want to hide them from people, and I have no issues with them, but I want to avoid having them in repository files in any way.